tests/models/whisper/test_modeling_whisper.py

```diff
@@ -1430,7 +1430,7 @@ def __init__(
         hidden_dropout_prob=0.1,
         attention_probs_dropout_prob=0.1,
         max_position_embeddings=20,
-        max_source_positions=30,
+        max_source_positions=1500,
```

Comment:
Hi @raghavanone @sanchit-gandhi 

I am wondering why we increase these values a lot ...?

Reply:
Whisper works intrinsically on a sequence length of 30s inputs (which corresponds to 1500 log mel spectrogram frames)

We could use a shorter context window (i.e. 30s -> 15s), we just need to reduce the feature extractor window length

Reply:
Fixed in https://github.com/huggingface/transformers/pull/24105